src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.12.6-preview",
+  "version": "1.0.0",
```

Comment:
@cormacpayne I need more information from you on the release out of preview. I met all the requirements in the swagger spec review (see the linked spec review). That is also why there are so many "breaking" changes and why this is being updated to 1.0.0. This is a HUGE change from our previous preview packages, which is why we wanted to get this in ASAP. As you mentioned, I will need to mitigate all the SDK breaking changes in PowerShell, so I need the necessary time to do so. I will address the update comments and regenerate, but for all breaking changes I will address that in PowerShell until the next round of allowed breaking changes when I will update everything to be flattened in PowerShell.